bundles/azure/src/main/java/org/apache/gravitino/abs/fs/AzureSasCredentialsProvider.java

```diff
@@ -0,0 +1,127 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.abs.fs;
+
+import java.io.IOException;
+import org.apache.gravitino.catalog.hadoop.fs.FileSystemUtils;
+import org.apache.gravitino.catalog.hadoop.fs.GravitinoFileSystemCredentialsProvider;
+import org.apache.gravitino.credential.ADLSTokenCredential;
+import org.apache.gravitino.credential.AzureAccountKeyCredential;
+import org.apache.gravitino.credential.Credential;
+import org.apache.hadoop.conf.Configurable;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.azurebfs.extensions.SASTokenProvider;
+
+public class AzureSasCredentialsProvider implements SASTokenProvider, Configurable {
```

Comment:
I want to check the name for the last time,  when I see `AzureSasCredentialsProvider` , I'm wondering is it a credential provider provided by Gravitino or is it a credential provider by original azure, I can't distinguish it from the name. I prefer to add Gravitino to the name, like `GravitinoAzureSasCredentialsProvider`,  WDYT? @jerryshao @yuqi1129

Reply:
Not only this interface, others like `FileSystemProvider`  also have interfaces with the same name,  based on your words, I think we need change the all classes that may conflict with third-party classes together. 

The class lies in model `gravitino-azure`, I can't get why this class could be a credential provider by original azure.

Besides,  there is NO `AzureSasCredentialsProvider` or any implementations of `SASTokenProvider`

Reply:
`FileSystemProvider` is different, there are  no similiar concept in cloud vendors. All and all, If you have strong option to keep current name, I'm ok.